pages/common/lynx.md

```diff
@@ -0,0 +1,23 @@
+# lynx
+
+> A Command-line web browser.
+
+- To visit a website:
+
+`lynx {{https://duckduckgo.com}}`
+
+- To move among the links:
```

Comment:
Perhaps this will read better?
```suggestion
- Navigate forwards and backwards through the links on a page:
```

Also, do <kbd>Insert</kbd> and <kbd>Delete</kbd> work for scrolling up and down? They do in `elinks`. If so, maybe we can add that here.

Reply:
No, `Insert` and `Delete` doesn't work for scrolling.